examples/packages/apollo-server-code-first/tsconfig.json

```diff
@@ -1,8 +1,9 @@
 {
-  "extends": "../../tsconfig.json",
   "compilerOptions": {
     "incremental": true,
     "composite": true,
+    "experimentalDecorators": true,
```

Comment:
do we need decorators?

Reply:
Yes. But seems only for `nestjs` and `type-graphql`. I will remove for others.